runtime/src/anchor.rs

```diff
@@ -100,11 +119,20 @@ decl_module! {
 
 
             let block_num = <system::Module<T>>::block_number();
-            <Anchors<T>>::insert(anchor_id, AnchorData {
+            let child_storage_key = common::child_storage_key(storage_days);
```

Comment:
the name `storage_days` is confusing here since it sounds relative to current date, will rename it to mean its the number of days from the epoch (non relative)